libs/common/src/database/schemas/users.schema.ts

```diff
@@ -72,6 +72,9 @@ export class User extends AbstractDocument {
 
   @Prop(String)
     referedBy: string
+
+  @Prop(Number)
```

Comment:
Make 0 default. It will throw and error and cause a crash if left null or undefined in DB.